Update documentation of user controls

LuisGuenther opened the pull request:
# Updating documentation of user controls

Issue: #3087 

## Description

**Documentation (screenshot) for user controls updated to also include custom views and suspicious metrics.**

ce-bo reviewed (commented):
Would you please add the new "link metrics" symbol? Thank you!

ce-bo commented:
In the Custom View documentation, there is the following text
`Custom Views allow you to save and upload your individual configurations for certain maps. To save new configurations there is a "shortcut-button". In order to be able to access more settings, there is another button next to the "shortcut-button" with the following functions:
`

I would suggest to update the term "shortcut-button" to "quick add button" and link it to the user-controls docs. Therefore, we need to separate the to Custom Views buttons in the user-controls-screenshot.

LuisGuenther commented:
Thank you for your feedback @ce-bo

LuisGuenther commented:
I added a header to indicate the enabled experimental features. I can later be removed by clipping the image when the features are officially made stable.
I also added explanations for the "link metrics" and the "custom view shortcut" buttons.

ce-bo commented on gh-pages/_docs/06-04-custom-view.md:
```diff
@@ -3,8 +3,10 @@ permalink: /docs/custom-view/
 ---
 
-Custom Views allow you to save and upload your individual configurations for certain maps. To save new configurations there is a "shortcut-button". In order to be able to access more settings, there is another button next to the "shortcut-button" with the following functions:
+Custom Views allow you to save and upload your individual configurations for certain maps. To save new configurations there is a [quick add button]({{site.baseurl}}{% link _docs/06-02-user-controls.md %}) in the menu bar. In order to be able to access more settings, there is another button next to the "shortcut-button" with the following functions:
```
```suggestion
Custom Views allow you to save and upload your individual configurations for certain maps. To save new configurations there is a [quick add button]({{site.baseurl}}{% link _docs/06-02-user-controls.md %}) in the menu bar. In order to be able to access more settings, there is another button next to the "quick add button" with the following functions:
```
